.circleci/config.yml

```diff
@@ -33,8 +35,32 @@ jobs:
           name: Verify if documentation is up-to-date
           command: npm run test:docs
 
+  deploy:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/repo
+      - run:
+          name: Authenticate with registry
+          command: echo "//registry.npmjs.org/:_authToken=$npmjs_token_cosmo" > ~/repo/.npmrc
+      - run:
+          name: Publish package
+          command: npm publish
+
 workflows:
   version: 2
-  build-test:
+  build-test-deploy:
     jobs:
-      - build-and-test
+      - build-test:
+          filters:
+            tags:
+              only: /.*/
+      - deploy:
+          requires:
+            - build-test
+          filters:
+            tags:
+              only: /^v.*/
+            branches:
```

Comment:
I think this is what we want per-version release and dev release, no?
            branches:
              only:
                - master